cloud/smartos/imgadm

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# © 2014, Hugo Silva <freebsd.consulting@gmail.com>
+
+DOCUMENTATION = '''
+---
+module: imgadm
+author: Hugo Silva
+version_added: 1.8.2
+short_description: Manage SmartOS VM images
+description:
+    -  The imgadm tool allows you to interact with virtual machine images on a
+       SmartOS system. Virtual machine images (also sometimes referred to as
+       'datasets') are snapshots of pre-installed virtual machines which are
+       prepared for generic and repeated deployments.
+options:
+    command:
```

Comment:
we try to avoid command as it is too imperative, also it does not seem to be needed as it can be inferred from the use of url vs uuid